### PR TITLE
inforna

### PR DESCRIPTION
**Pull request by HassanAmr**
removed inforna from the backlist

### CODE REVIEW

File: tools/rna_tools/inforna/inforna.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<tool id="inforna" name="InfoRNA" version="0.1.0">
+<tool id="inforna" name="InfoRNA" version="2.1.2">
     <description></description>
     <requirements>
-        <requirement type="package" version="1.0">inforna</requirement>
+        <requirement type="package" version="2.1.2">inforna</requirement>
         <requirement type="package" version="2.1.5">vienna_rna</requirement>
```
Comment (bgruening) [inline tools/rna_tools/inforna/inforna.xml]: vienna_rna is not needed with conda as separate dependency https://github.com/bioconda/bioconda-recipes/blob/master/recipes/inforna/meta.yaml#L21

Comment (HassanAmr) [inline tools/rna_tools/inforna/inforna.xml]: Ok, I will remove it.
I still need to make some test cases for this one. But  I'm checking if it will work.

Comment (HassanAmr) [inline tools/rna_tools/inforna/inforna.xml]: Is it correct that I remove tool_dependencies.xml?

**Comment by bgruening** [comment]
Yeah, tests needs to be defind.

**Comment by HassanAmr** [comment]
Hi @bgruening, can you please help me with this. I try to test inforna on my linux machine using the following:
```
INFO-RNA "(((((...(((((......))))).)))))" -c "NNNNNUGCNNNNNCAGUGHNNNNNCNNNNN" -v "000001100000000000100000000000" -n 999 -R 1 -N 2 -Fm -p 0.1
```

Which I got from the test data from http://rna.informatik.uni-freiburg.de/INFORNA/Result.jsp?toolName=INFORNA&jobID=7154744

And my output is as the following:
```
=========================
Initializing Step: 
=========================
Wanted Structure    : (((((...(((((......))))).)))))
Sequence Constraints: NNNNNUGCNNNNNCAGUGHNNNNNCNNNNN
Allowed Mismatches  : 000001100000000000100000000000
Designed Sequence   : GGGCCUGCGCCCCCAGUGAGGGGCCGGCCC
designed randomly   : 0

=========================
Local Search Results: 
=========================
MFE:    GGGCCUUCGCCCCCAGUGAGGGGCCGGCCC    1  (-19.90)
number of mismatches: 1
```
Which doesn't matcht the output I got from the online test

**Comment by bgruening** [comment]
This is probably due to a version difference and should not confuse you.

**Comment by bgruening** [comment]
Thanks!

**Comment by HassanAmr** [comment]
My pleasure :)

**Pull request merged**